contributing/documentation/standards.rst

```diff
@@ -7,8 +7,14 @@ look and feel familiar, you should follow these standards.
 Sphinx
 ------
 
-* The following characters are chosen for different heading levels: level 1
-  is ``=``, level 2 ``-``, level 3 ``~``, level 4 ``.`` and level 5 ``"``;
+* The following characters are chosen for different heading levels:
+
+    * level 1 ``=``;
+    * level 2 ``-``;
+    * level 3 ``~``;
+    * level 4 ``.``;
+    * level 5 ``"``;
```

Comment:
I wouldn't use a vertical list here, I think a vertical list is good enough (we should keep things as short as possible)